appveyor.yml

```diff
@@ -66,11 +66,10 @@ test_script:
 
       if ($testsExitCode -ne 0) {exit $testsExitCode}
 
-  - ps: |
-      & coverage erase
+  - coverage erase
+  - del /Q coverage.xml
 
-      $clcachePy = "$((Get-Location).Path)\clcache.py"
-      $env:CLCACHE_CMD = "coverage run -a --branch --include=$clcachePy $clcachePy";
+  - ps: |
```

Comment:
Kinda funny how this is really the core of it all: don't run 'coverage' recursively...